[레벨 3 글쓰기 미션] 찰리(박민영) 미션 제출합니다.

Gomding opened the pull request:
부족한 글에 솔직한 리뷰 마구 써주세요!!
팀 프로젝트 하면서 쌓인 스트레스 리뷰로 풀기 ➡️ [$$스트레스$$무료풀기##](https://github.com/Gomding/woowa-writing-3/blob/level3/LEVEL3.md)

@puterism
@soulgchoi
@Rok93
@sjpark-dev
@joyykim
@thisisyoungbin

Gomding committed:
docs(LEVEL1.md) : Level 1 글쓰기

Gomding committed:
글쓰기 미션 1단계 작성

Gomding committed:
LEVEL1.md 수정

Gomding committed:
gather 의 장점을 리스트화

Gomding committed:
Update LEVEL1.md

Gomding committed:
Update LEVEL1.md

Gomding committed:
미션이 있는것은 -> 미션이 있다는 것은 수정

Gomding committed:
과거의 약속들이 어떻게 에너지를 소모시켰는지 구체화

Gomding committed:
Code With Me 의 단점에 대한 예시 추가

Gomding committed:
오타수정

Gomding committed:
게더의 장점을 추가

Gomding committed:
모호한 표현 수정

Gomding committed:
Code With Me 에서 겪었던 문제를 비유를 통해 설명

Gomding committed:
Update LEVEL1.md

Gomding committed:
Update LEVEL1.md

Gomding committed:
지쳐버렸습니다 소주제 마지막 부분의 어색한 부분 수정

Gomding committed:
Update LEVEL1.md

Gomding committed:
전사팀 -> 전사교육팀 으로 수정

Gomding committed:
글쓰기 시작

Gomding committed:
글쓰기 레벨2 작성 완료 (맞춤법 /문법검사까지)

Gomding committed:
스타일 적용 안된곳 수정

Gomding committed:
오류 나는 부분 수정

Gomding committed:
이해 했을수도 -> 이해했을 수도 / 이해못했을 수도 -> 이해 못했을 수도 변경

Gomding committed:
우테코에 오기 전 학습법은 정말 단순합니다. -> 단순했습니다. 로 변경

Gomding committed:
이미 앞에서 언급한 내용은 간단하게 나타내도록 변환

Gomding committed:
이유가 모호한 문장에 이유를 추가

Gomding committed:
문장 수정

또한 블로그에 학습 내용이 저장되는 게 아닌 머리속에 학습 내용이 정리되어 저장됐습니다.

->

블로그에 단순히 학습 내용을 옮겨적는게 아니라 머리속에 정리된 내용들을 적을 수 있었습니다.

Gomding committed:
html 문법 제거하기

Gomding committed:
docs(LEVEL3.md) : Level 3 글쓰기

Hyeon9mak commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+굉장히 공감하는 것이 팀원간에 친밀할 수록 의견을 내는것도 자유롭고 의견에 반대하는것도 편하다.   
+그렇기에 우리에겐 개발얘기도 중요하지만 드라마 이야기, 사는 이야기 등등 흔히 말하는 '썰 풀기'는 정말 중요하다고 생각한다.   
+백기선님의 영상에서 이런 말씀을 하셨다.    
+"의견 충돌은 인간 대 인간이 아니라 의견 대 의견이 충돌하고 있다고 생각해라, 말싸움에서 이기는 것이 내 가치를 증명하는 것이 아니다."   
+팀원 간 서로가 친밀하다면 어떤 사람인지 더욱 잘 알고있을 것이고 '의견이 충돌하고 있다' 에 더욱 집중할 것이다.
+오늘도 가볍게 누군가는 말할 수 있다. "우리 오늘 올림픽 양궁 결승전인데 잠깐보고 할까요?"
+
```
👍

Gomding committed:
맞춤법 수정

unluckyjung commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+1000명의 사람이 있다면 1000가지의 다른 환경에서 자라왔을 것이다.   
+그래서 모든 사람이 100% 같은 성향을 가질 확률은 없을 것이다. 그 때문에 충돌은 당연하다.   
+예전에 책에서 이런 글을 읽었다.    
+"세상에 나랑 같은 사람만 존재한다면 이 세상은 너무나 재미없을 것이다."      
```
ㄹㅇㅋㅋ

DWL5 replied:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋ

sjpark-dev commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+Microsoft의 개발 문화가 꼭 좋다고 할 수 없다. 하지만 믿고 들어볼 가치는 있다고 생각한다.   
+Microsoft의 개발자 문화는 엄청 좋았다. 아니 감동할 정도라고 생각한다.   
+   
+팀 프로젝트의 경험을 말하기 전에 Microsoft의 개발자 문화 중 감명 깊었던 몇 가지를 짧게 소개하려고 하려고 한다.   
```
```suggestion
팀 프로젝트의 경험을 말하기 전에 Microsoft의 개발자 문화 중 감명 깊었던 몇 가지를 짧게 소개하려고 한다.   
```

매의 눈 👀

Gomding replied:
매의 눈 너무나 감사합니다 😃

sjpark-dev commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+    + 자신이 주는 임팩트를 이해하면 동기부여가 되고 집중도 되고 시야가 넓어진다.
+    + 지금 하는 일이 큰 그림의 서비스에서 어디에 해당하는지 이해하고 어떻게 협력을 하는지 보인다면 의사소통에 더욱 도움이 될 것이다.
+
+이 외에 여러 가지 내용이 있지만, 협업에관한 내용만 간단하게 소개를 했다.   
```
```suggestion
이 외에 여러 가지 내용이 있지만, 협업에 관한 내용만 간단하게 소개를 했다.   
```

Gomding replied:
와 정말 잘 찾아주시는 군요. 감사합니다 🙇

sjpark-dev commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+    + 디테일하게는 몰라도 어떻게 흘러가는지 정도는 알고 있어야 한다.
+* 용기와 실수
+    + 실수는 누구나 저지를 수 있고 실수를 받아들일 수 있어야 한다.
+    + 실수를 받아들일 수 있어야 자기의 생각을 공개적으로 말할 수 있다.
```
정말 좋은 말이네요 👍
실수를 두려워하지 않는 용기 정말 중요하다고 생각해요.

Joyykim replied:
완전 공감합니다!! 😀

sjpark-dev commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+(개인적인 생각일 수도 있다..)
+틀린 의견 속에서도 어떤 키워드가 트리거가 되어 좋은 의견으로 바뀔 수 있고   
+본인은 틀린 의견이라 생각해도 다른 사람이 그 의견을 보완해서 좋은 의견이 될 수 있다.   
+하지만 무엇보다 중요한 건 팀 전체가 이런 분위기를 만들어야 한다. 그런 의미에서 놀토라는 팀은 의견을 내기 정말 좋은 분위기다.      
```
`놀토라는 팀은 ~ 분위기다.`

주어 서술어 일치!

Gomding replied:
감사합니다!! 3번째 매의눈 🦅

sjpark-dev commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+팀 내에서 회의 때마다 자주 나오는 말이 있다.   
+"나 뭔가 생각난 게 있어.. 그런데 아.. 아니야"   
+이때 우리는 항상 같은 대답을 한다.   
+#### "제발 말해줘 궁금해서 오늘 잠 못 잘 것 같아"
```
팀 분위기 정말 좋은 것 같아요. 😎

Gomding replied:
한번 준줌방 소회의실로 놀러오시죠 😎

sjpark-dev commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+나는 예전에 인터넷에서 이런 글을 본 적이 있다. 오래전에 본 글이라 실제 내용과 차이는 있을 수 있다.      
+한 학생이 아이패드를 사기 위해서 아이패드를 사면 자신에게 어떤 점에서 도움이 되는지 PPT를 만들어 부모님에게 발표를 한 것이다.      
+결말은 학생이 아이패드를 얻고 해피엔딩으로 끝난다.   
+가장 쉬운 설득 방법은 새로운 기술에 대한 지식을 확실하게 쌓고 새로운 기술이 우리 작업에서 어떤 도움을 줄 수 있고 현재보다 어떤 점에서 더 좋은지 등을 발표하는 것이다.      
```
긴 호흡의 문장이네요.
쉼표를 사용해서 나누거나 복수의 문장으로 나누는 것은 어떨까요?

Rok93 replied:
에드의 생각과 비슷해요!  
```suggestion
가장 쉬운 설득 방법은 먼저 새로운 기술에 대한 지식을 확실하게 쌓는 것이다. 그리고 새로운 기술이 우리 작업에서 어떤 도움을 줄 수 있고 현재보다 어떤 점에서 더 좋은지 등을 발표하는 것이다.      
``` 
음... 부족한 점이 많겠지만 이렇게라도 한번 문장을 쉬어가게 하면 어떨까요?

sjpark-dev commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+"세상에 나랑 같은 사람만 존재한다면 이 세상은 너무나 재미없을 것이다."      
+나는 이 말에 적극 공감한다. 나는 말하는 것보다 듣는 것을 좋아한다.      
+아마 나랑 같은 사람만 존재한다면 세상은 고요할 것이다.   
+세상에 다양한 사람들이 있다는 것에 감사하며 즐거움을 최대한 만끽하자
```
```suggestion
세상에 다양한 사람들이 있다는 것에 감사하며 즐거움을 최대한 만끽하자.
```

`.` 마침표 깜빡하셨네요. 😆
혹은 '!' 느낌표였을까요?

Rok93 replied:
느낌표가 좋을 것 같네요 😃

Gomding replied:
좋은 의견 감사합니다. ! 로 마무리 너무 깔끔하네요!

sjpark-dev reviewed (commented):
안녕하세요 찰리! 🙂

팀 프로젝트를 하기 전에 읽으면 정말 유익할 것 같은 글 잘 읽었어요.

몇 가지 코멘트 남겼는데 개인적인 의견일 뿐이니 참고만 해주세요.

thisisyoungbin commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+'의견을 말하지 않으면 손해' 분위기 어떻게 만들어 갈까? 라고 물어본다.   
+이에 대답은 간단하게 말할 수 있다. '의견 편하게 말하라고 하면 되는 거 아니에요?'   
+여기에 추가로 '틀린 의견이라 생각해도 일단 편하게 말해보자'라는 것이다.   
+이번 프로젝트를 하면서 나는 의견을 말하는 것 보다는 충분히 듣고 말하도록 노력해봤다.   
```
것 보다는 -> 것보다는

Gomding replied:
욘-하 엄청 오랜만인것 같네.. 루터가고싶다 😭 

피드백 즉시 반영 👍

thisisyoungbin commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+굉장히 공감하는 것이 팀원 간에 친밀할수록 의견을 내는 것도 자유롭고 의견에 반대하는 것도 편하다.   
+그렇기에 우리에겐 개발 얘기도 중요하지만, 드라마 이야기, 사는 이야기 등등 흔히 말하는 '썰 풀기'는 정말 중요하다고 생각한다.   
+백기선 님의 영상에서 이런 말씀을 하셨다.    
+"의견 충돌은 인간 대 인간이 아니라 의견 대 의견이 충돌하고 있다고 생각해라, 말싸움에서 이기는 것이 내 가치를 증명하는 것이 아니다."   
```
멋있는 말이네요

Rok93 replied:
오... 이말은 많은 영감을 주는 말이네요.

thisisyoungbin reviewed (commented):
찰-하
역시 글을 잘써서 간단하게 피드백 몇 개 남겼습니당!

sihyung92 commented:
세상에 찰리 같은 사람만 있으면 넘 재밌을거같은디 ^^ 
글 잘 읽고 갑니다 의견은 부수되 사람은 부수지말자!! (?)

Joyykim commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+    + 실수를 받아들일 수 있어야 자기의 생각을 공개적으로 말할 수 있다.
+    + '자신의 의견을 말하지 않으면 팀에게 손해'라는 분위기를 만드는 것이 중요!
+    + 틀린 아이디어도 용납해줘라
+* 사용자에게 우리가 하는 일이 비즈니스 임팩트가 있어야 한다.
```
비즈니스 임팩트? 우리가 개발하는게 실제 비즈니스에 어떤 효과를 가져다 주는지 이해하는게 중요하다는 건가요? 재밌는 내용이네요😀

Joyykim replied:
제가 잘 몰라서 그런걸까요? `비즈니스 임팩트`에 대한 설명이 있으면 좋을 것 같네요ㅎㅎ 개인적인 생각임다~

Gomding replied:
실제 비즈니스를 생각하시면 됩니다!
물론 직접적이 아니라 간접적인것도 포함입니다.
예를들면 우리가 자동화를 통해 다른 팀의 업무가 더욱 빠르게 진행할 수 있게 만들어준다면
결과적으로 영향을 받은 팀이 회사에 비즈니스 임팩트를 주기때문에
이런 과정들을 이해하는 것으로 더욱 동기부여를 하는것이죠! 물론 일에 대한 이해도 증가하겠죠 :)
설명을 조금 넣는게 확실히 좋아보이네요 :)

Joyykim commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+한 사람을 설득하기 위해서는 그만큼의 노력이 필요하다. 쉽게 설득하고 그것이 받아들여진다 해서 상대방이 깊게 공감하고 함께 열정적으로 협업할 수 있는가?      
+진심으로 설득이 됐다면 새로운 기술을 도입하다 문제가 생겼을 때 발 벗고 나서서 도와주지 않겠는가?   
+그런 의미에서 나는 가능하면 팀 전원이 공감한 상태에서 새로운 기술을 도입하고 싶다.   
+설득하기 위한 공부 재밌지 않은가?   
```
멋져요!!!!!!!!!!!!!!!!!!!!!!!

Puterism replied:
오 좋은 마인드입니다!

Joyykim commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+그렇기에 우리에겐 개발 얘기도 중요하지만, 드라마 이야기, 사는 이야기 등등 흔히 말하는 '썰 풀기'는 정말 중요하다고 생각한다.   
+백기선 님의 영상에서 이런 말씀을 하셨다.    
+"의견 충돌은 인간 대 인간이 아니라 의견 대 의견이 충돌하고 있다고 생각해라, 말싸움에서 이기는 것이 내 가치를 증명하는 것이 아니다."   
+팀원 간 서로가 친밀하다면 어떤 사람인지 더욱 잘 알고 있을 것이고 '사람이 아닌 의견이 충돌하고 있다.' 에 더욱 집중할 것이다.
```
우테코에서 협업을 하면서 매번 느끼는 점이죠ㅎㅎ

Rok93 commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+    + 실수를 받아들일 수 있어야 자기의 생각을 공개적으로 말할 수 있다.
+    + '자신의 의견을 말하지 않으면 팀에게 손해'라는 분위기를 만드는 것이 중요!
+    + 틀린 아이디어도 용납해줘라
+* 사용자에게 우리가 하는 일이 비즈니스 임팩트가 있어야 한다.
```
개인적인 의견인데 순서를 이렇게 바꾸면 어떨까요? 
```suggestion
* 우리가 하는 일이 사용자에게 비즈니스 임팩트가 있어야 한다.
```

Gomding replied:
😮 이렇게 바꾸면 더 부드럽게 읽어지네요. 언어의 마술사 이신가요?

Rok93 commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+
+## 내가 감동한 Microsoft의 개발자 문화
+인생은 타이밍인가 "어떻게 하면 현명하게 팀 프로젝트를 진행할 수 있을까?"라고 고민할 때 굉장히 좋은 라이브 방송을 접했다.   
+백기선 님이 "개발자로 살아남는 방법"이라는 타이틀로 Microsoft의 개발자 문화를 공유해준다.   
```
아니... 기선님 영상 잘 챙겨보는데... 이렇게 좋은 방송을 해주셨다니... 😂

Gomding replied:
유튜브 구독 월 1900원..
다시보기 가능..

Rok93 replied:
이미 구독중입니다. 👀 👍
기선님 제 선장님이 되어주시겠습니까? 🥀 

![image](https://user-images.githubusercontent.com/47850258/128640270-29becae7-fe02-4307-9aea-4888af325900.png)

Rok93 commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+    + 지금 하는 일이 큰 그림의 서비스에서 어디에 해당하는지 이해하고 어떻게 협력을 하는지 보인다면 의사소통에 더욱 도움이 될 것이다.
+
+이 외에 여러 가지 내용이 있지만, 협업에관한 내용만 간단하게 소개를 했다.   
+특히 용기와 실수에서 '자신의 의견을 말하지 않으면 팀에게 손해라는 분위기를 만드는 것이 중요' 라는 항목이 감동적이었다.   
```
이 분위기 만드는게 중요하다는 이야기는 정말 많이 들었는데, 그렇게 만드는게 어려운 것 같아요 🤔

Gomding replied:
저도 노력은 하지만 정말 어렵다고 생각해요!!
하지만 분위기를 만든다면 엄청난 파워라고 생각해요.

Rok93 commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+이에 대답은 간단하게 말할 수 있다. '의견 편하게 말하라고 하면 되는 거 아니에요?'   
+여기에 추가로 '틀린 의견이라 생각해도 일단 편하게 말해보자'라는 것이다.   
+이번 프로젝트를 하면서 나는 의견을 말하는 것 보다는 충분히 듣고 말하도록 노력해봤다.   
+(개인적인 생각일 수도 있다..)
```
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 아마도 찰리의 생각대로 팀원들도 느끼고 있을거에요 😃

Gomding replied:
감사합니당 ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

Rok93 commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+팀 내에서 회의 때마다 자주 나오는 말이 있다.   
+"나 뭔가 생각난 게 있어.. 그런데 아.. 아니야"   
+이때 우리는 항상 같은 대답을 한다.   
+#### "제발 말해줘 궁금해서 오늘 잠 못 잘 것 같아"
```
어.... 이 스킬 줍줍 해갑니다.  고마워요 찰리~~ 🙏

Puterism replied:
오 좋은 방법이네요 꿀팁 알아갑니다

Gomding replied:
자매품으로 ```오늘 한 명은 불면증으로 잠을 못잡니다``` 도 있습니다. 😄

Rok93 commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+하지만 어떤 사람은 생전 처음 듣는 기술을 보고 쉽게 공감하기는 어려울 것이다.   
+"왜 그 기술을 써야 하지요?"
+나는 예전에 인터넷에서 이런 글을 본 적이 있다. 오래전에 본 글이라 실제 내용과 차이는 있을 수 있다.      
+한 학생이 아이패드를 사기 위해서 아이패드를 사면 자신에게 어떤 점에서 도움이 되는지 PPT를 만들어 부모님에게 발표를 한 것이다.      
+결말은 학생이 아이패드를 얻고 해피엔딩으로 끝난다.   
```
앗... 이거 기억나요 !!  난 왜 때만썼지 했었던 기억이 ㅋㅋㅋ 🥲

Rok93 commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+1000명의 사람이 있다면 1000가지의 다른 환경에서 자라왔을 것이다.   
+그래서 모든 사람이 100% 같은 성향을 가질 확률은 없을 것이다. 그 때문에 충돌은 당연하다.   
+예전에 책에서 이런 글을 읽었다.    
+"세상에 나랑 같은 사람만 존재한다면 이 세상은 너무나 재미없을 것이다."      
```
하... 잠시 상상했는데... 벌써 질리네요 😫

Gomding replied:
로키만 있으면 재밌을거같아요!

Rok93 reviewed (approved):
찰리 글 잘 읽었어요!!  👍
팀프로젝트를 통해서 느끼고 계신 감정들 공감도 많이 됐고, 또 제가 생각지도 못했던 부분들에 대해서도 많은 깨달음을 얻으신 것 같아요. 😃

전 사실 글 읽으면서 뭔가 피드백 드릴 부분들 보다는 찰리가  팀 프로젝트에서 느꼈던 감정들을 제 경험들과 빗대어서 생각해보게 됐던 것 같아요. 그래서인지 딱히 피드백드릴 부분이 안보이네요 🥲 
에드가 진짜 매의 눈 👀 이라고 느껴질 정도로 피드백을 잘해줬네요... 

조금 이른감이 없지않지만 전 지금의 찰리의 글도 좋다고 생각돼서 approve 할게요 👍
남은 팀 프로젝트도 잘 하실거라 믿어요 💪

Joyykim commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+한 학생이 아이패드를 사기 위해서 아이패드를 사면 자신에게 어떤 점에서 도움이 되는지 PPT를 만들어 부모님에게 발표를 한 것이다.      
+결말은 학생이 아이패드를 얻고 해피엔딩으로 끝난다.   
+가장 쉬운 설득 방법은 새로운 기술에 대한 지식을 확실하게 쌓고 새로운 기술이 우리 작업에서 어떤 도움을 줄 수 있고 현재보다 어떤 점에서 더 좋은지 등을 발표하는 것이다.      
+한 사람을 설득하기 위해서는 그만큼의 노력이 필요하다. 쉽게 설득하고 그것이 받아들여진다 해서 상대방이 깊게 공감하고 함께 열정적으로 협업할 수 있는가?      
```
> 쉽게 설득하고 그것이 받아들여진다 해서 상대방이 깊게 공감하고 함께 열정적으로 협업할 수 있는가?

이 문장의 의미가 한번에 이해하기 힘든 것 같아요. 좀 더 다듬어 볼 수 있을까요?

Joyykim reviewed (approved):
잘봤습니다 찰리 ^^ 너무 잘 쓰셔서 피드백 드릴만한게 없네요! 찰리가 얼마나 프로젝트를 위해 고민하고 노력하고 있는지 알 수 있었어요ㅎㅎ 읽으면서 힐링되었네요 감사해요~ 수고하셨습니다!!
남은 프로젝트도 화이팅👍👍👍

Puterism commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+## 잡담은 곧 팀워크다.
+글을 쓰는 지금에서 얼마 전 유튜브에서 ["배달의민족 CEO에게 뽑고 싶은 개발자를 물어보았다"](https://www.youtube.com/watch?v=3H4umWD5bwI&t=515s) 라는 영상을 봤다.   
+이 영상에서 '송파구에서 일 잘하는 11가지 방법' 중  "잡담은 경쟁력이다"라는 한가지 문구를 말씀하셨다.   
+굉장히 공감하는 것이 팀원 간에 친밀할수록 의견을 내는 것도 자유롭고 의견에 반대하는 것도 편하다.   
```
저도 이 영상을 봤었는데, 정말 많은 인사이트를 얻을 수 있었어요
잡담은 정말로 경쟁력이 맞는 것 같아요
팀원 간 친밀도가 높을수록 전체적으로 프로젝트가 더 잘 진행되고 으쌰으쌰하는 게 생긴다고 생각해요

Puterism commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+## 내가 감동한 Microsoft의 개발자 문화
+인생은 타이밍인가 "어떻게 하면 현명하게 팀 프로젝트를 진행할 수 있을까?"라고 고민할 때 굉장히 좋은 라이브 방송을 접했다.   
+백기선 님이 "개발자로 살아남는 방법"이라는 타이틀로 Microsoft의 개발자 문화를 공유해준다.   
+Microsoft의 개발 문화가 꼭 좋다고 할 수 없다. 하지만 믿고 들어볼 가치는 있다고 생각한다.   
+Microsoft의 개발자 문화는 엄청 좋았다. 아니 감동할 정도라고 생각한다.   
```
```suggestion
백기선님이 소개해주신 Microsoft의 개발 문화는 엄청 인상 깊었다. 아니 감동할 정도라고 생각한다.
```

`Microsoft의 개발 문화가 꼭 좋다고 할 수 없다. 하지만 믿고 들어볼 가치는 있다고 생각한다.` 라는 문장이 문맥과는 맞지 않아서 불필요한 문장이라는 생각이 들어서, 이를 지우고 아랫문장을 조금 다듬어서 제안드립니다!

Puterism reviewed (approved):
찰리 안녕하세요!
다른 곳에서 얻은 인사이트를 지금 진행하고 있는 프로젝트에도 적용해보려는 시도가 인상 깊은 글이었습니다
남은 프로젝트 기간도 화이팅입니다 💪 

피드백 하나 남겼는데 참고 부탁드릴게요 :)

365kim commented:
![image](https://user-images.githubusercontent.com/60066472/128603970-ba63dbbf-ea22-415b-8387-39861d385b1e.png)

찰리 이거 센스 너무 좋으신데욬ㅋㅋㅋ 센스 배워감니당 👍👍👍

soulgchoi commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+팀 내에서 회의 때마다 자주 나오는 말이 있다.   
+"나 뭔가 생각난 게 있어.. 그런데 아.. 아니야"   
+이때 우리는 항상 같은 대답을 한다.   
+#### "제발 말해줘 궁금해서 오늘 잠 못 잘 것 같아"
```
본적 없지만 귓가에서 바로 재생되고요?🤣

soulgchoi commented on LEVEL3.md:
```diff
@@ -0,0 +1,82 @@
+한 학생이 아이패드를 사기 위해서 아이패드를 사면 자신에게 어떤 점에서 도움이 되는지 PPT를 만들어 부모님에게 발표를 한 것이다.      
+결말은 학생이 아이패드를 얻고 해피엔딩으로 끝난다.   
+가장 쉬운 설득 방법은 새로운 기술에 대한 지식을 확실하게 쌓고 새로운 기술이 우리 작업에서 어떤 도움을 줄 수 있고 현재보다 어떤 점에서 더 좋은지 등을 발표하는 것이다.      
+한 사람을 설득하기 위해서는 그만큼의 노력이 필요하다. 쉽게 설득하고 그것이 받아들여진다 해서 상대방이 깊게 공감하고 함께 열정적으로 협업할 수 있는가?      
+진심으로 설득이 됐다면 새로운 기술을 도입하다 문제가 생겼을 때 발 벗고 나서서 도와주지 않겠는가?   
```
이 부분 정말 좋네요. 쉽게 번 돈은 쉽게 쓰듯이(비유가 이래서 죄송...) 쉬운 설득은 그만큼 쉽게 여겨질 수 있을 수 있다는...!

Gomding replied:
일단 쉽게 돈 버는 법을 알려주세요.. 😢

soulgchoi reviewed (approved):
안녕하세요, 찰리. 글 잘 읽었어요~
찰리는 항상 텍스트에서 위트가 느껴지는 사람 같아요. 글 정말 재밌었어요. 팀 프로젝트를 진심으로 즐기고 있음을 알 수 있는 글이었습니다!
백기선님 라이브 시간이 맞으면 종종 봤는데 이렇게 글로 정리해줘서 고마워요. 정말 주옥같은 말씀들이 많았네요. 이 중에서 대화와 관련된 부분을 본인 프로젝트에 대한 내용으로 잘 끌고가고 있어서 좋았어요. 마침표나 띄어쓰기는 다른 크루들이 잘 피드백 해주어서 저는 이만..
마지막까지 즐거운 프로젝트가 되길 바래요~

thisisyoungbin reviewed (approved):
찰-하! 더운데 잘 지내고 있지~?

찰리의 센스가 돋보이는 글 아주 잘 읽었습니다ㅎㅎ
백기선님 개발자로 살아남는 방법 나중에 봐야 할 todo list에 놓고 아직 못봤는데 찰리 덕분에 예습도 하고 좋네요 :)
자유롭게 의견을 낼 수 있는 팀 분위기가 형성되어 있어 좋은 팀문화가 정착이 되었군요 ! 또 충돌을 당연하게 생각하고 받아들이는 부분이 인상적이었습니다. 

개인적으로 멀티태스킹이 잘 안되는 편이라 `팀에서 하는 업무에 관심을 가져야 한다` 는 부분이 많이 어렵더라구요. 그래서 궁금한 게 있으면 나중에 물어보게 됐는데 항상 팀원들이 잘 알려줘서 고마운 마음을 갖고 있습니다 ㅎㅎ

피할 수 없다면 즐기는? 자세로 충돌을 즐겨봐야겠네요.
글 잘 읽었습니다 찰리!! 남은 기간도 화이팅~~~~~~~~~~~~~~~

Gomding committed:
Merge branch 'gomding' into level3

Gomding committed:
피드백 반영

- ```개발자가 하는 일은 어떤 방식으로든 비즈니스 임팩트를 주게 되어있다.``` 에 대한 간단한 설명 추가
- ```소개하려고 하려고 한다.``` -> 소개하려고 한다.
- ```협업에관한``` -> 협업에 관한
- ```가장 쉬운 설득 방법은 새로운 기술에 대한 지식을 확실하게 쌓고 새로운 기술이 우리 작업에서 어떤 도움을 줄 수 있고 현재보다 어떤 점에서 더 좋은지 등을 발표하는 것이다.``` -> 문장 나누기
    + ```가장 쉬운 설득 방법은 먼저 새로운 기술에 대한 지식을 확실하게 쌓는 것이다. 그리고 새로운 기술이 우리 작업에서 어떤 도움을 줄 수 있고 현재보다 어떤 점에서 더 좋은지 등을 발표하는 것이다.```

woowahan-pjs reviewed (approved):
지금의 경험이 다음 프로젝트에 분명 도움이 될 거예요. 고생 많았어요! 👍
